### PR TITLE
fix: PHPCS fixes according to latest ruleset.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
Phpcs fixes

### CODE REVIEW

File: classes/class-astra-widgets-helper.php [hunk 1]
```diff
@@ -426,7 +426,7 @@ function generate( $self, $fields = array(), $repeater_id = '' ) {
 											data-field-id="<?php echo esc_attr( $value['id'] ); ?>"
 											data-icon-visible="<?php echo esc_attr( ( isset( $value['show_icon'] ) ) ? $value['show_icon'] : 'no' ); ?>"
 										/>
-										<span><?php echo $value['desc']; ?></span>
+										<span><?php echo esc_attr( $value['desc'] ); ?></span>
```
Comment (Balachandark) [inline classes/class-astra-widgets-helper.php]: This needs to be esc_html

File: classes/class-astra-widgets-helper.php [hunk 2]
```diff
@@ -445,7 +445,7 @@ function generate( $self, $fields = array(), $repeater_id = '' ) {
 											<?php checked( $value['default'] ); ?>
 											id="<?php echo esc_attr( $self->get_field_id( $value['id'] ) ); ?>"
 											name="<?php echo esc_attr( $self->get_field_name( $value['id'] ) ); ?>" />
-										<label for="<?php echo esc_attr( $self->get_field_id( $value['id'] ) ); ?>"><?php echo $value['name']; ?></label>
+										<label for="<?php echo esc_attr( $self->get_field_id( $value['id'] ) ); ?>"><?php echo esc_attr( $value['name'] ); ?></label>
```
Comment (UttamSharma007) [inline classes/class-astra-widgets-helper.php]: esc_html instead of esc_attr.

File: classes/class-astra-widgets-helper.php [hunk 3]
```diff
@@ -567,15 +567,15 @@ class="astra-field-image-preview-id"
 								$field_name = $self->get_field_name( $value['id'] );
 							}
 							?>
-								<div class="astra-widget-field astra-widget-field-select astra-widget-field-<?php echo $value['id']; ?>">
+								<div class="astra-widget-field astra-widget-field-select astra-widget-field-<?php echo esc_attr( $value['id'] ); ?>">
 									<div class="astra-widget-field-<?php echo esc_attr( $value['id'] ); ?>">
-									<label for="<?php echo esc_attr( $field_id ); ?>"><?php echo $value['name']; ?></label>
+									<label for="<?php echo esc_attr( $field_id ); ?>"><?php echo esc_attr( $value['name'] ); ?></label>
```
Comment (UttamSharma007) [inline classes/class-astra-widgets-helper.php]: esc_html instead of esc_attr.

File: classes/widgets/class-astra-widget-social-profiles.php [hunk 4]
```diff
@@ -332,9 +329,9 @@ function widget( $args, $instance ) {
 			$title            = apply_filters( 'widget_title', $this->get_fields( 'title' ) );
 
 			// Before Widget.
-			echo $args['before_widget'];
+			echo $args['before_widget'];// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
 			if ( $title ) {
-				echo $args['before_title'] . $title . $args['after_title'];
+				echo $args['before_title'] . esc_html( $title ) . $args['after_title'];// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```
Comment (Balachandark) [inline classes/widgets/class-astra-widget-social-profiles.php]: $title should not have esc_html

**Pull request merged**